lib/inputs/rhel/static.ts

```diff
@@ -0,0 +1,27 @@
+import { ExtractAction, ExtractedLayers } from "../../extractor/types";
+import { streamToJson } from "../../stream-utils";
+
+export const getRedHatReposContentAction: ExtractAction = {
+  actionName: "redhat-content-manifests",
+  filePathMatches: (filePath) => isRedHatContentManifest(filePath),
+  callback: streamToJson,
```

Comment:
I believe you could already here add a type here that would only return us object like: 
```
{
  layerIndex: number,
  repositories: [ ]
}
```
and use `streamToJson<T>` as a callback:
https://github.com/snyk/snyk-docker-plugin/blob/0026c770b4dea73feff4a9547fcbee653006d2f6/lib/stream-utils.ts#L69

Then we already have it handled and don't have to return whole json :-)

Reply:
Then we don't also have to worry about stuff we don't need in tests

Reply:
Leaving an FYI note - we discussed offline, not applicable currently. Adding the type only